tempesta_fw/http_sess_conf.h

```diff
@@ -24,6 +24,8 @@
 #include "cfg.h"
 #include "http_types.h"
 
+extern const unsigned short tfw_cfg_redirect_st_code_dflt;
+
```

Comment:
Maybe just use #define or `static const short` instead of exporting the single integer?